pycheribuild/projects/disk_image.py

```diff
@@ -1069,6 +1069,223 @@ def cheribsd_class(self):
+            "bc",
+            "dc",
+            "netstat",
+            "*ftp*",
+            "telnet",
+            "objcopy",
+            "strip",
+            "yacc",
+            "sysstat",
+            "edit",
+            "ee",
+            "ree",
+            "asn1_compile",
+            "Mail",
+            "mail*",
+            "nm",
+            "vacation",
+            "addr3line",
+            "sftp",
+            "kadmin",
+            "truss",
+            "bsnmp*",
+            "rpcgen",
+            "top",
+            "localdef",
+            "drill",
+            "hxtool",
+            "cu",
+            "tip",
+            "patch",
+            "m4",
+            "calendar",
+            "dialog",
+            "mkimp",
+            "kdump",
+            "diff",
+            "slc",
+            "tftp",
+            "indent",
+            "iscsictl",
+            "ar",
+            "ranlib",
+            "lpr",
+            "bsdiff",
+            "vmstat",
+            "kcc",
+            "klist",
+            "kswitch",
+            "lpq",
+            "users",
+            "unifdef",
+            "lprm",
+            "pr",
+            "primes",
+            "crunchgen",
+            "vtfontcvt",
+            "gprof",
+            "finger",
+            "ssh-agent",
+            "syscall_timing",
+            "nc",
+            "diff3",
+            "bsdcpio",
+            "nfsstat",
+            "host",
+            "rpcinfo",
+            "elfdump",
+            "at",
+            "atq",
+            "atrm",
+            "batch",
+            "hd",
+            "hexdump",
+            "od",
+            "sockstat"
+        ]
+        self.mtree.exclude_matching(list(map(lambda p: "./usr/bin/" + p, usr_bin_globs)))
+
+        usr_lib_exceptions = [
+            "pam_*",
+            "libgnuregex.so.5",
+            "libdevinfo.so.6",
+            "libbz2.so.4",
+            "libpam.so.6",
+            "libbsm.so.3",
+            "libprivatessh.so.5",
+            "libblacklist.so.0",
+            "libgssapi_krb5.so.10",
+            "libgssapi.so.10",
+            "libkrb5.so.11",
+            "libwrap.so.6",
+            "libprivateldns.so.5",
+            "libroken.so.11",
+            "libopie.so.8",
+            "libasn1.so.11",
+            "libypclnt.so.4",
+            "libcom_err.so.5",
+            "libhx509.so.11",
+            "libwind.so.11",
+            "libheimbase.so.11",
+            "libprivateheimipcc.so.11",
+            "libssl.so.111"
+        ]
+        self.mtree.exclude_matching("./usr/lib/*", list(map(lambda p: "./usr/lib/" + p, usr_lib_exceptions)))
+
+        self.mtree.exclude_matching("./usr/libexec/*", ["*/getty"])
+
+        usr_sbin_exceptions = [
+            "service",
+            "sshd",
+            "newsyslog",
+            "utx",
+            "syslogd"
+        ]
+        self.mtree.exclude_matching("./usr/sbin/*", list(map(lambda p: "./usr/sbin/" + p, usr_sbin_exceptions)))
```

Comment:
Much less of a problem here, but maybe removing all and then explicitly adding the 5 binaries is makes this code easier to understand? Some of the arrays are positive, others are negative.